easybuild/easyconfigs/s/scikit-bio/scikit-bio-0.5.9-foss-2023a.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-bio'
+version = '0.5.9'
+local_commit = '7565847'
```

Comment:
This commit doesn't correspond to the `0.5.9` release at all, I've opened https://github.com/easybuilders/easybuild-easyconfigs/pull/20300 to rectify that...